python/src/aiconfig/editor/server/server.py

```diff
@@ -350,6 +350,10 @@ def kill_thread(thread_id: int | None):
             yield "["
             yield json.dumps({"aiconfig": aiconfig_json})
             yield "]"
+        
+        yield "["
+        yield json.dumps({"stop_streaming": None})
+        yield "]"
```

Comment:
Oh, interesting. I didn't realize each streamed message was being wrapped in `[` and `]` separately -- just noting the original intention was for the full streamed response to be a single array of JSON objects ([ marking the start, ] marking the end). I guess if it works ¯\_(ツ)_/¯

Reply:
Yea I actually tried doing the entire thing, and that works for single instance, but when we're iterating through the stream queue, we need to reset every yield around individual `[` and `]` otherwise it will not update on client